js/islandora_book_reader.js

```diff
@@ -306,6 +306,16 @@
         $('#BookReader').find(icon).attr('title', titles[icon]);
       }
     }
+
+    var booktitle
+     = this.bookTitle;
+    // Add alt tag for screen reader accessibility.
+    $('.BRnoselect').each(function(i) {
+      // Gives each book page a title, reminiscent of
+      // 'Sample Book: Page 1'.
+      var title = booktitle + ": " + Drupal.t("Page") + " " + (i + 1);
```

Comment:
@MorganDawe @Islandora/7-x-1-x-committers @adam-vessey i have a comment on this. Sadly this escaped my review yesterday. This is technically correct. It gives each page a title + a sequential page and by doing so it "conforms" with wcag, but really really this is wrong. I mean this labels the front cover of a book as page 1? I feel this is not right, just adds a "fake" compliance by generating wrong metadata (or metadata based UX). Either we make this based on each page label (object label) or we used something different than page that speaks more about the sequence order things are being displayed here and not so much about the semantics of a book and its pages..constituents.

Reply:
Thank you very much for getting another look at this @DiegoPino, i am absolutely open to modifying this as the community see's fit. Including the page label is a great idea, however when it comes to screen readers i am afraid it would not convey necessary information for, say, the visually impaired. How do you feel about a combination of the two, perhaps something like:

{Obj label}: {X} of {Y} ({Page obj label})

So a real world example could be:

Great expectations: 5 of 250 (Great Expectations.pdf-5)

I do get what you are saying tho, the 'page' number this would be adding may not line up with the 'pages' in the book. Like not taking into account the table of contents (if one exists) in the source document, nor do blank pages at the beginning of the source, or the end of the document, etc. Bit of a black hole of 'what ifs'.

I cant really think of a one size fits all solution, beyond what i have suggested above. Without any significant preprocessing, do you have any suggestions?

Reply:
@DiegoPino I'm afraid @MorganDawe  is out this week, he'll get back to this as soon as he can.